### DISCUSSION TITLE
Connect-PnPOnline Interactive approach not working which uses username and password

### DISCUSSION THREAD

**Discussion by PriyankaGontla** [pnp/powershell#4264] [Q&A]
I'm getting the following error when trying to athenticate using connect-pnponline -interactive and using a username and password, client ID/Secret and cert still work fine.

There have been no changes to the Enterprise Registration so i'm unsure as to where the issue lays

AADSTS700016: Application with identifier '31359c7f-bd7e-475c-86db-fdb8c937548e' was not found in the directory

This is happening on all version of PNP we have in our org, 1.12.0 and 2.12.0

**Comment by gautamdsheth** [comment]
Hello, please see this:

https://github.com/pnp/powershell/discussions/4249
